uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/components/manager-leave-calendar.js

```diff
@@ -111,7 +111,7 @@ define([
      * @param  {Array} leaveRequests - leave requests array from API
      */
     vm._indexLeaveRequests = function (leaveRequests) {
-      vm.leaveRequests = {};
+      vm.leaveRequests = [];
```

Comment:
This is more confusing to be honest, i was sure that `vm.leaveRequest = []` was wrong. You are doing
```js
vm.leaveRequests[leaveRequest.contact_id] = // something
```
how can `vm.leaveRequest` be an array?